nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/DepthToSpace.java

```diff
@@ -66,6 +67,13 @@ public DepthToSpace(INDArray in, INDArray out, int blockSize, String dataFormat)
         addIArgument(blockSize, isNHWC ? 1 : 0);
     }
 
+    public DepthToSpace(INDArray x, int blockSize, DataFormat dataFormat) {
+        super(null, x,null,null,null);
+        this.blockSize = blockSize;
+        this.dataFormat = dataFormat.toString();
+
+    }
```

Comment:
You're not setting iargs here, that's necessary.

or better yet, just use the existing constructor to avoid code duplication:
```
    public DepthToSpace(INDArray x, int blockSize, DataFormat dataFormat) {
        this(x, null, blockSize, dataFormat);
    }
```